cmd/clusterctl/client/config/providers_client.go

```diff
@@ -108,7 +108,8 @@ const (
 
 // Add-on providers.
 const (
-	HelmAddonProviderName = "helm"
+	HelmAddonProviderName  = "helm"
+	FleetAddonProviderName = "fleet"
```

Comment:
I think we have to use a org prefix: https://cluster-api.sigs.k8s.io/developer/providers/contracts/clusterctl#adding-a-provider-to-clusterctl

Given that rancher-sandbox & rancher are both owned by rancher, I think `rancher-fleet` is fine according to our policy